### DISCUSSION TITLE
Disable on screen controller Android

### DISCUSSION THREAD

**Discussion by guymanley** [diasurgical/DevilutionX#3409] [Q&A] [answered]
Hello. Loving this port. Been playing the vita vpk for a long while. I pulled out my Android based console on a whim and thought I'd give this a try. Runs great. Is there a way to disable on screen controller? Excuse if I'm being ignorant of something obvious. But I looked through the ini, wiki, issues and discussions and didn't find an answer.

Retroid pocket 2 Lineage OS Android 8.1 Running release 1.3.0
![20211106_163027](https://user-images.githubusercontent.com/50853425/140623678-bd0f88f8-5d22-432d-a5de-c7daab8219fb.jpg)

**Comment by Trihedraf** [comment, marked as answer]
Not yet unfortunately. I believe the next version is the goal for the option to be added.

**Reply by guymanley** [reply]
Thank you. I'll look forward to it. This fits so well with this device 4:3 both modern and nostalgic. Runs great and the controller support is so much easier to work with then the 1st vita build from 2020. 👍

**Reply by AJenbo** [reply]
for now you can get around it by placing 5 empty png files in a folder called `ui_art` next to the MPQ files:
`button.png`  `directions2.png`  `directions.png`  `menu-levelup.png`  `menu.png`

**Reply by guymanley** [reply]
Thank you. Though if potions are on my belt they also float on screen LOL. But this is completely fine until and option to toggle screen controls is available. Here's another photo. 🙂
![20211107_045601](https://user-images.githubusercontent.com/50853425/140640503-fb99f430-1140-4dbc-9e2b-a0a03e44c29a.jpg)

**Reply by PGuitarnut** [reply]
This does not work for me...  Whenever I make some "blank png" and insert them in a new folder called ui_art the touch controls remain.  If I make a 1 pixel white png the touch controls turn into white boxes.
Please explain exactly, step by step, how to achieve this.  I have tried using Paint, and Photoshop  with no luck.

**Reply by PGuitarnut** [reply]
> for now you can get around it by placing 5 empty png files in a folder called `ui_art` next to the MPQ files: `button.png` `directions2.png` `directions.png` `menu-levelup.png` `menu.png`

How is this done exactly ?

**Reply by AJenbo** [reply]
Did you try transparent?

**Reply by AJenbo** [reply]
Considering that both of those graphics normally has transparent areas I would say that it should work correctly a fully transparent image. It also appear to have worked for `button.png`.

**Reply by PGuitarnut** [reply]
OK my mistake was including the ".png" in the names.
In photoshop:
- Open any image
- Create New Layer
- Drag New Layer to the bottom (so it becomes background)
- Erase the original Layer
- Save as PNG and make copies
- Rename them as described above without the ".png" extension
- Create ui_art folder in (device)\Internal shared storage\Android\data\org.diasurgical.devilutionx\files\
- Drag all the "transparent png" in there

Now it works and looks just like your picture :)  Thanks

**Reply by AJenbo** [reply]
This is probably because you have extensions hidden on your computer so the names actually became .png.png

**Reply by qndel** [reply]
*sends dolphin.jpg.exe*

**Comment by AJenbo** [comment]
Thanks for sharing the picture :)

**Reply by ebenegui** [reply]
@AJenbo is there a .png related to potions to also hide those?
I removed all other touch controls successfully with the workaround but I'm also seeing the floating potions as well.
Thanks!

**Reply by qndel** [reply]
no

**Reply by AJenbo** [reply]
It's a hack, not an intentional feature. We will work on having it work properly automatic in the next version. At which point we suggest you remove your png files.
